.github/workflows/zowe-cli.yml

```diff
@@ -57,6 +57,10 @@ jobs:
       if: ${{ always() && steps.build.outcome == 'success' }}
       run: npm run test:unit >> file.txt
 
+    - name: Unlock Keyring
+      if: ${{ always() && steps.build.outcome == 'success' && matrix.os == 'ubuntu-latest' }}
+      uses: t1m0thyj/unlock-keyring@master
```

Comment:
@t1m0thyj 
Will this cause problems to the `nektos/act` scripts?
Maybe we can add an ID and skip/remove this step all together in the [testCliWorkflow.js](https://github.com/zowe/zowe-cli/blob/next/scripts/testCliWorkflow.js)